Shared/BetterColorPicker.cs

```diff
@@ -0,0 +1,137 @@
+﻿using System;
+using BepInEx;
+using BepInEx.Configuration;
+using ChaCustom;
+using HarmonyLib;
+using Illusion.Component.UI.ColorPicker;
+using IllusionUtility.GetUtility;
+using KKAPI.Utilities;
+using TMPro;
+using UnityEngine;
+using UnityEngine.UI;
+
+using PluginInfo = BetterColorPicker.Info;
+
+namespace BetterColorPicker
+{
+    [BepInDependency(KKAPI.KoikatuAPI.GUID)]
+    [BepInPlugin(PluginInfo.GUID, "Better Color Picker", PluginInfo.Version)]
+    [BepInProcess(Constants.MainGameProcessName)]
```

Comment:
Process filters for studio and the steam version are missing for KK.

Reply:
Added version to BepInDependency.